stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/internal/ChannelViewHolder.kt

```diff
@@ -223,15 +225,14 @@ internal class ChannelViewHolder @JvmOverloads constructor(
     private fun configureForeground(diff: ChannelListPayloadDiff, channelItem: ChannelListItem.ChannelItem) {
         binding.itemForegroundView.apply {
             diff.run {
+                val lastMessage = channelItem.channel.getLastMessage()
                 if (nameChanged || (channelItem.channel.isAnonymousChannel() && diff.usersChanged)) {
-                    configureChannelNameLabel()
+                    configureChannelNameLabel(lastMessage)
                 }
 
                 if (avatarViewChanged) {
                     configureAvatarView()
                 }
-
-                val lastMessage = channelItem.channel.getLastMessage()
                 if (lastMessageChanged || typingUsersChanged) {
                     lastMessageLabel.isVisible = channelItem.typingUsers.isEmpty() && lastMessage.isNotNull()
```

Comment:
If a user is typing on a channels that doesn't contain any message, the title is center but it shouldn't because on that case it is overlapping with the typing indicator